gcp/notebook_setup/README.md

```diff
@@ -14,17 +14,17 @@ This section describes the process required to:
 ... processing ...
 Packing conda environment
 Collecting packages...
-Packing environment at '[CONDA ENV]/rapids0.19_py3.7' to 'rapids0.19_py3.7.tar.gz'
+Packing environment at '[CONDA ENV]/rapids21.06_py3.7' to 'rapids21.06_py3.7.tar.gz'
```

Comment:
```suggestion
Packing environment at '[CONDA ENV]/rapids21.06_py3.8' to 'rapids21.06_py3.8.tar.gz'
```
Since you're using py3.8 below to manually pack the env, I'll also update the helper script